torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -117,21 +117,38 @@ def check_node_safe(node: Node):
     The test suite test_aot_autograd_cache.py::AOTAutogradCachePicklerTests tries its best to fully cover/specify this behavior.
     """
     SAFE_TORCH_MODULES = ("torch.functional", "torch.nn.functional")
+    SAFE_TORCH_FUNCTIONS = (
```

Comment:
it might be worth a comment that this list only exists because we are being very conservative and assuming that arbitrary `torch.*` API's are not cache safe unless we mark them as such (just to call out that there is nothing special about these particular 6 functions). wdyt?